common/constants/default.go

```diff
@@ -166,4 +166,6 @@ const (
 
 	EdgeNodeRoleKey   = "node-role.kubernetes.io/edge"
 	EdgeNodeRoleValue = ""
+
+	DeafultMosquittoContianerName = "mqtt-kubeedge"
```

Comment:
```suggestion
	DeafultMosquittoContainerName = "mqtt-kubeedge"
```

Reply:
Seems haven't used ?

Reply:
> Seems haven't used ?

message resource and name used it, for example:
![image](https://github.com/kubeedge/kubeedge/assets/48788150/722e537e-7c6e-4082-8e12-cdfc9eadd4b0)

Reply:
OK, so please just the typo